### DISCUSSION TITLE
Europe/Ukraine VoWiFi works with lifecell

### DISCUSSION THREAD

**Discussion by andrii-z** [kyujin-cho/pixel-volte-patch#209] [Carrier-specific Experiences]
Phone: Google Pixel 6a
Carrier: lifecell (formerly **life:)**)
VoLTE: almost always works
VoWiFi: works

**Comment by andrii-z** [comment]
Phone: Google Pixel 8
Carrier: lifecell (formerly life:))
VoLTE: almost always works
VoWiFi: works

**Reply by decadance** [reply]
hello, how u bypass activation via lifecell website? 
![Screenshot_20240117-222320](https://github.com/kyujin-cho/pixel-volte-patch/assets/5555924/5532a1b9-d228-42a1-a760-0763c21580d1)

**Reply by andrii-z** [reply]
I did it with a supported phone (iPhone). I think, unfortunately, this is the only way now.
Check the device list on the carrier's website. Maybe a friend or family member has one of these.

**Reply by fillwithjoy1** [reply]
Wait, so VoLTE works on Pixel 8 but not Pixel 6?

**Reply by andrii-z** [reply]
> Wait, so VoLTE works on Pixel 8 but not Pixel 6?

It looks like it depends on the carrier's base station, or there are other reasons. I tested Pixel 6a for a very short time. VoLTE worked only a few times. I've been testing Pixel 8 longer, almost always the VoLTE worked fine. But it didn't work in the same places where it didn't work on Pixel 6a.

**Reply by fillwithjoy1** [reply]
that's very weird. I'm planning to add this carrier to the supported carriers list (see #258), but you indeed have some weird issues

**Reply by andrii-z** [reply]
Yeah, VoLTE works very weird. But VoWiFi works well.

**Comment by decadance** [comment]
Doing this way too, I thought u have another solution, but it's fine,
thanks 👍

**Comment by andrii-z** [comment]
Pixel 8. voWiFi stopped working after the June 2024 feature drop/security update 😭
